### PR TITLE
Create shatanik.json

### PR DESCRIPTION
**Pull request by shatanikmahanty**
Add data for registration

## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/shatanik.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Shatanik's Developer Portfolio",
+  "owner": {
+    "username": "shatanikmahanty",
+    "email": "shatanikmahanty@gmail.com"
+  },
+  "record": {
+    "CNAME": "shatanikmahanty.github.io/shatanikmahanty"
```
Comment (DEV-DIBSTER) [inline domains/shatanik.json:8]: Invalid CNAME record, it should just be a hostname without the / and additional characters.

**Review by DEV-DIBSTER — DISMISSED** [review]

**Review by triach-rold — CHANGES_REQUESTED** [review]
404. Unreachable.

**Comment by shatanikmahanty** [comment]
> shatanikmahanty.github.io

Will check the docs again. Actually for me it was available at shatanikmahanty.github.io/shatanikmahanty

File: domains/shatanik.json (hunk 1 above)
Comment (shatanikmahanty) [reply domains/shatanik.json]: Thanks for the review!

**Comment by CuteDog5695** [comment]
> shatanikmahanty.github.io/shatanikmahanty

oh thats fine then

**Review by iostpa — APPROVED** [review]
seems good to me now

**Pull request merged**